src/redux/features/profile.feature.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  gender: '',
+  language: '',
+  currency: '',
+  location: '',
+  departement: '',
+  manager: '',
+  birthdate: '',
+};
+
+export const profileSlice = createSlice({
+  name: 'profile',
+  initialState: {
+    value: initialState,
+  },
+  reducers: {
+    updatedProfile: (state, action) => {
+      state.value = action.payload;
+    },
```

Comment:
Create actions that will be dispatched when updating fails or when it's pending instead of using state from other features.